### ISSUE TITLE
[Drive] provide Download ability

### ISSUE THREAD

**Issue by ildar** [tobykurien/GoogleApps#31]
sometimes people share files via G Drive. Provided a link this app can show some info about the files but doesn't show the "Download" button. I hope that it can be fixed and users would be able to download files with this app.

**Comment by WalidHella** [comment]
I agree. Without a Download/Upload option, the Drive shortcut is almost useless.
